app/liquid_tags/spotify_tag.rb

```diff
@@ -29,7 +32,7 @@ def render(_context)
   private
 
   def parse_uri(uri)
-    URI_REGEXP.match(uri) || raise_error
+    Regexp.union(URI_REGEXP, URI_PLAYLIST_REGEXP).match(uri) || raise_error
```

Comment:
didn't know about `Regexp.union`, so cool!

Reply:
Me neither 😂 everything in Ruby is new to me so making this PR took more time than I expected.